pkgs/development/node-packages/default.nix

```diff
@@ -40,11 +39,12 @@ let
         wrapProgram $out/bin/aws-azure-login \
             --set PUPPETEER_EXECUTABLE_PATH ${pkgs.chromium}/bin/chromium
       '';
-    };
+      meta = oldAttrs.meta // { platforms = pkgs.lib.platforms.linux; };
+    });
 
-    bitwarden-cli = super."@bitwarden/cli".override (drv: {
-      name = "bitwarden-cli-${drv.version}";
-      meta.mainProgram = "bw";
+    bitwarden-cli = super."@bitwarden/cli".override (oldAttrs: {
+      name = "bitwarden-cli";
```

Comment:
Can we use pname here?

Reply:
It seems the `pname` isn't defined, so that's not and option:

```console
~/C/nixpkgs on nodePackages-mainProgram
❯ nix eval -f . nodePackages."@bitwarden/cli".pname
error: attribute 'pname' in selection path 'nodePackages.@bitwarden/cli.pname' not found
       Did you mean name?
```